translations/CONTRIBUTING.hindi.md

```diff
@@ -0,0 +1,151 @@
+# हसुरा ग्राफक्यूएल-इंजन में योगदान
+
+_पहले_: यदि आप योगदान शुरू करने में असुरक्षित महसूस करते हैं, तो बेझिझक हमसे पूछें हमारे [डिस्कोर्ड चैनल](https://discordapp.com/invite/hasura) पर #contrib चैनल में या Marion को DM करें (@marion#9554)। आप अपने योगदान के साथ आगे भी बढ़ सकते हैं और हम आपको प्रतिपुष्टि देंगे। चिंता न करें - सबसे बुरा यह हो सकता है कि आपको विनम्रता से कुछ बदलने के लिए कहा जाएगा। हम किसी भी योगदान की सराहना करते हैं, और हम नहीं चाहते उसके बीच नियमों की एक दीवार खड़ी हो।
```

Comment:
> या Marion को DM करें (@marion#9554)

@marionschleifer Is it okay for people to DM you on discord to get started with the initial contribution. Or should this line be removed?